crates/core/src/generator/trait.rs

```diff
@@ -470,8 +470,9 @@ where
                             let mut req = req.to_owned();
                             req.args = Some(args);
 
-                            if fuzz_tx_value.is_some() {
-                                req.value = fuzz_tx_value;
+                            if let Some(fuzz_tx_value_unwrapped) = fuzz_tx_value {
+                                req.value =
+                                    Some(fuzz_tx_value_unwrapped.parse().unwrap_or_default());
```

Comment:
we should allow an error to crash here (use `?`) instead of calling `unwrap_or_default` -- if we try to parse an invalid value, we want to fail and let the user know what to fix, so they're not confused later when their transaction has value = 0.

I'd add a new variant `ParseValueFailed(...)` to [`GeneratorError`](https://github.com/flashbots/contender/blob/main/crates/core/src/generator/error.rs) and use `map_err` to map errors from `parse` to `ParseValueFailed(...)`. In the contents of the variant (...) ideally we'd use `#[from] ParseError` to interpret the parse error directly. But a String may also suffice.